packages/atlas/src/views/viewer/AuctionView/AuctionView.tsx

```diff
@@ -0,0 +1,339 @@
+                      1508 blocks
+                    </Text>
+                    <Information
+                      text="Auctions are run and settled on-chain and use blocks of operations rather than clock time."
+                      footer={<Text variant="t100">Auctions closing block: 123115234</Text>}
+                      placement="top"
+                    />
+                  </FlexWrapper>
+                </FlexWrapper>
+              </Header>
+              {type === 'auction' ? (
+                <>
+                  <CurrentBidWrapper>
+                    {BID ? (
+                      <ActiveBidWrapper>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Current bid
+                          </Text>
+                          <FlexWrapper>
+                            <CurrentBidAvatar size="bid" />
+                            <BidderName variant="h400">{BID.bidder.name}</BidderName>
+                          </FlexWrapper>
+                        </ActionBarCell>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Bid amount
+                          </Text>
+                          <FlexWrapper>
+                            <CurrentBidJoyToken size={24} variant="silver" />
+                            <Text variant="h400">{BID.amount}</Text>
+                          </FlexWrapper>
+                        </ActionBarCell>
+                      </ActiveBidWrapper>
+                    ) : (
+                      <ActiveBidWrapper>
+                        <ActionBarCell>
+                          <Text variant="h300" secondary spacing={{ bottom: 2 }}>
+                            Current bid
+                          </Text>
+                          <Text variant="h400">Nobody has bid yet</Text>
+                        </ActionBarCell>
+                      </ActiveBidWrapper>
+                    )}
+                  </CurrentBidWrapper>
+                  <MinimumBidWrapper>
+                    <MinimumBid>
+                      <Text variant="h300" secondary>
+                        Minimum bid:
+                      </Text>
+                      <JoyTokenIcon variant="silver" size={24} /> <Text variant="h400">{MINIMUM_BID}</Text>
+                    </MinimumBid>
+                    <div>
+                      <Text variant="t100" secondary>
+                        Buy now: {FIXED_PRICE} tJOY
+                      </Text>
+                    </div>
+                  </MinimumBidWrapper>
+                  <TextField
+                    placeholder={`Min. ${MINIMUM_BID} tJOY`}
+                    nodeStart={<JoyTokenIcon variant="silver" size={24} />}
+                    nodeEnd={!!placedBid && <Pill variant="overlay" label={`$${convertToUSD(placedBid)}`} />}
```

Comment:
We need to round this number, the current precision is way too precise, I think 2 numbers for fractional part is max. We should also handle really low values do we don't display `0.00` if the value is `0.004`. @TCzechowski any ideas here?